workflow-report/README.md

```diff
@@ -0,0 +1,44 @@
+# Github Action: report a failed job to VK Team chat
+This action sends the following information to VK Team chat:
+* __Job__: name of the failed job with the link to this job
+* __Commit__: hash of the commit started the job with the link to this commit
+* __Branch__: name of the branch with the link to this branch
+* __History__: the link to commit history
+* __Triggered on__: name of the event triggered the workflow
+* __Committer__: GitHub login of the commit author
+* __Commit message subject__: subject (first string) of the commit message
+* __Failed steps info__: info about the failed steps (outputs, outcome, conclusion)
+
+# Prepare your workflow
+Action can tell you exact steps that failed only if your steps have ID, example:
+```yaml
+steps:
+  - name: This step does some useful work
+    id: some-useful-work
+    run: some useful work
+
+  - name: This step does some other useful work
+    id: other-useful-work
+    uses: some/additional/action@v1.0
+    run: other useful work
+```
+
+# Usage
+All the fields are required.
+```yaml
+      - name: Send VK Teams notification
+        if: failure()
+        uses: tarantool/actions/workflow-report@master
+        with:
+          api-url: ${{ secrets.VKTEAMS_BOT_API }}
+          bot-token: ${{ secrets.VKTEAMS_BOT_TOKEN }}
+          chat-id: ${{ secrets.VKTEAMS_DEBUG_CHAT_ID }}
+          job-steps: ${{ ToJson(steps) }}
+```
+| variable  | how to get                                    | example                              |
+|-----------|-----------------------------------------------|--------------------------------------|
+| api-url   | `/help` in Metabot                            | https://myteam.mail.ru/bot/v1        |
+| bot-token | You get when create a bot in Metabot          | 000.1234567890.0987654321:1111111111 |                            
+| chat-id   | In the chat info                              | @tntcore_ghaction_chat               |
+| job-steps | Please, set exactly as `${{ ToJson(steps) }}` | ${{ ToJson(steps) }}                 |
```

Comment:
If it must not change, why allow to change it at all?

Reply:
Workflow have to send steps to action. I tried to call this inside action, that didn't work